packages/common/src/utils/SafeLocalStorage.ts

```diff
@@ -24,23 +34,29 @@ export const SafeLocalStorage = {
     value: SafeLocalStorageItems[Key]
   ): void {
     if (isSafe()) {
-      localStorage.setItem(key, JSON.stringify(value))
+      if (typeof value === 'string') {
+        localStorage.setItem(key, value)
+      } else {
+        localStorage.setItem(key, JSON.stringify(value))
+      }
     }
   },
-  getItem<Key extends keyof SafeLocalStorageItems>(key: Key): SafeLocalStorageItems[Key] | null {
+  getItem<Key extends keyof SafeLocalStorageItems>(
+    key: Key
+  ): SafeLocalStorageItems[Key] | undefined {
     if (isSafe()) {
       const value = localStorage.getItem(key)
 
       if (value) {
         try {
           return JSON.parse(value)
         } catch {
-          return value
+          return undefined
```

Comment:
Nope, checked everthing we read. We are trying to use undefined for falsy values instead of `null`. I remember we've been discussed this long ago (maybe last year). So for such values we should avoid `null` and use `undefined` (not sure what is the strong why)

Reply:
I prefer to have `undefined` personally as it prevents you from doing `| null` everywhere else so ok for me